internal/pkg/auth/user_login.go

```diff
@@ -60,22 +65,31 @@ func AuthorizeUser(p *print.Printer, isReauthentication bool) error {
 		}
 	}
 
-	listener, err := net.Listen("tcp", ":0")
-	if err != nil {
-		return fmt.Errorf("bind port for login redirect: %w", err)
+	var redirectURL string
+	var listener net.Listener
+	var listenerErr error
+	var port int
+	for i := range configuredPortRange {
+		port = defaultPort + i
```

Comment:
I think we could add a Debug log at the start saying: `trying to bind port X for login redirect`

This way it also clarifies the debug log below `"unable to bind port %d for login redirect: %s"` by making it clear that we are trying different ports and it is not a failure (at least yet)